docs/4.2/admin-guide.md

```diff
@@ -603,13 +597,11 @@ hardware keys as a second authentication factor. By default U2F is disabled. To
 start using U2F:
 
 * Enable U2F in Teleport configuration `/etc/teleport.yaml` .
-* For CLI-based logins you have to install
-
-  [u2f-host](https://developers.yubico.com/libu2f-host/) utility.
 
-* For web-based logins you have to use Google Chrome, as it is the only browser
+* For CLI-based logins you have to install [u2f-host](https://developers.yubico.com/libu2f-host/) utility.
 
-  supporting U2F at this time.
+* For web-based logins you have to use Google Chrome and Firefox 67, are the only 
```

Comment:
We should probably say Firefox 67 or greater, as they're now on version 71/72 I think.